pkg/storage/unified/apistore/prepare.go

```diff
@@ -81,7 +81,7 @@ func (s *Storage) prepareObjectForUpdate(ctx context.Context, updateObject runti
 		return nil, err
 	}
 	obj.SetOriginInfo(origin)
-	obj.SetUpdatedBy(user.GetUID().String())
+	obj.SetUpdatedBy(user.GetUID())
```

Comment:
@Jguer @gamab -- this is the most visible change.  k8s requires "GetUID()" to be a string, and we currently return a typed flavor